### ISSUE TITLE
setup.py 설정 문의드립니다.

### ISSUE THREAD

**Issue by KOR-PAIR** [AI-GrandChallenge/round-1#32]
아래 사용하는 패키지이며, CUDA는 10.1, Tensorflw-gpu는 1.9 사용합니다.
import operator
import tensorflow as tf
import numpy as np
import os
import scipy.io
import glob
import scipy.misc as misc
from imageio import imread
from PIL import Image
import time
import shutil
import random
from datetime import datetime, timedelta
import pandas as pd
import skimage
import skimage.io
import skimage.transform
import pandas as pd

**Comment by junsukchoe** [comment]
안녕하세요.

사용하시는 패키지들이 설치되어 있는 도커 이미지를 docker hub에 업로드 하신 후, 해당 주소를 setup.py 상단부에 적어주시면 됩니다! 도커 이미지를 만드신 후 혹시 추가적인 패키지가 있으실 경우 도커 이미지를 새로 만드실 필요 없이 해당 패키지의 이름을 `install_requires` 안에 추가하시면 됩니다. 

다음은 예시입니다:
(`tensorflow/tensorflow:1.9.0-gpu-py3` 이미지 사용, `tqdm`, `pillow`를 추가 설치할 경우)
```
#nsml: tensorflow/tensorflow:1.9.0-gpu-py3
from distutils.core import setup

setup(
    name='iitp_trash',
    version='1.0',
    install_requires=[
                      'tqdm',
                      'pillow',
                      ]
)

```

**Comment by kw-nisp** [comment]
tensorflow 1.9 버전이 아닌 다른버전을 사용하려 할때는 다음과 같이 변경을 하여 사용하면 되는 건가요?

#nsml: tensorflow/tensorflow:1.15.0-gpu-py3

**Comment by nsml-admin** [comment]
@kw-nisp 네 setup.py에 말씀하신대로 적어주시면 됩니다 (`tensorflow/tensorflow:1.15.0-gpu-py3` 이미지가 docker hub에 있다는 가정하에)

```python
#nsml: tensorflow/tensorflow:1.15.0-gpu-py3
from distutils.core import setup

setup(
    name='iitp_trash',
    version='1.0',
    install_requires=[
                      'tqdm',
                      'pillow',
                      ]
)
```
